frontend/babel.config.js

```diff
@@ -2,6 +2,7 @@ module.exports = function (api) {
   api.cache(true);
   return {
     presets: ["babel-preset-expo"],
-    plugins: ["module:react-native-dotenv"],
+    plugins: ["module:react-native-dotenv", "react-native-paper/babel"],
```

Comment:
we should remove react-native-dotenv here, it causes the welcome to expo screen of doom